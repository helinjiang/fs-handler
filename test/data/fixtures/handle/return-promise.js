export default Promise.resolve({
  name: 'return-promise',
  age: 16
});