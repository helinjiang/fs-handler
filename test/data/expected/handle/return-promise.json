{"name":"return-promise","age":16}
